portmap/core/views.py

```diff
@@ -48,17 +47,17 @@ def _get_index_context():
     query_structure = Article.get_query_structure()
     query_form = QueryIndexForm(data=None, datatypes=query_structure.keys())
     feedback_form = UseCaseFeedbackForm(data=None, datatype='None', source='', destination='')
-    datatype_help = GithubClient().get_datatype_help()
 
-    def create_datatype(name):
+    def create_datatype(datatype_object):
+        name = datatype_object.name
         return {
             'id': '_'.join(name.split()),
```

Comment:
I can follow up with a PR to start using actual IDs here; there's some JS that will need to be updated too